packages/devtools/lib/src/timeline/timeline_model.dart

```diff
@@ -298,55 +293,11 @@ class TimelineEvent {
 
   bool get isGpuEvent => type == TimelineEventType.gpu;
 
-  bool get isUiEventFlow => containsChildWithCondition(
-      (TimelineEvent event) => event.name.contains('Engine::BeginFrame'));
-
-  bool get isGpuEventFlow => containsChildWithCondition(
-      (TimelineEvent event) => event.name.contains('PipelineConsume'));
-
-  /// Depth of this TimelineEvent tree, including [this].
-  ///
-  /// We assume that TimelineEvent nodes are not modified after the first time
-  /// [depth] is accessed. We would need to clear the cache if this was
-  /// supported.
-  int get depth {
-    if (_depth != 0) {
-      return _depth;
-    }
-    for (TimelineEvent child in children) {
-      _depth = max(_depth, child.depth);
-    }
-    return _depth = _depth + 1;
-  }
-
-  int _depth = 0;
+  bool get isUiEventFlow => containsChildWithCondition((TreeNode node) =>
```

Comment:
you should be able to remove this cast as well.

Reply:
done